client/rest-high-level/src/main/java/org/elasticsearch/client/dataframe/transforms/DataFrameTransformConfig.java

```diff
@@ -48,6 +54,8 @@
     private final DestConfig dest;
     private final PivotConfig pivotConfig;
     private final String description;
+    private final Version transformVersion;
+    private final Date createTime;
```

Comment:
As this is new code, should we use the new java time classes (I guess `Instant` in this case)?

Reply:
@dimitris-athanasiou is this a thing? I did not know this was a thing. TIL.

Reply:
Yes, use `Instant` and serialize it using `StreamOutput.writeInstant()` and `StreamInput.readInstant()`.